lib/conditionals.md

```diff
@@ -5,8 +5,9 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a command that depends upon the result of another command to be run, ex. when, if, elsif, else, unless, etc.
```

Comment:
Look into `true` and `false`. What is their meaning in the context of a conditional?